Detections/SigninLogs/Brute Force Attack against GitHub Account.yaml

```diff
@@ -39,5 +39,5 @@ query: |
     | summarize FailedLoginsCountInRunTime = count() by User = Identity 
   ) on User 
   | where FailedLoginsCountInRunTime > LearningThreshold
-  | extend AccountCustomEntity = UserPrincipalName, IPCustomEntity = IPAddress
+  //| extend AccountCustomEntity = UserPrincipalName, IPCustomEntity = IPAddress
```

Comment:
Is entity extraction supposed to be commented ? 
I didn`t see any instructions when to use or not use in description

Reply:
Yeah, this is broken so commented it out while the author fixes it.